packages/rsocket-core/package.json

```diff
@@ -14,6 +14,7 @@
     "test": "yarn jest"
   },
   "devDependencies": {
+    "@types/jest": "^27.0.3",
```

Comment:
`jest` and `mocha` have conflicting type definitions that cause issues when compiling with `karma-typescript`, which we can overcome by moving the `@types/jest` into the dependencies list of the individual packages, and telling yarn workspaces not to house the types as well so that they are not discovered by typescript when compiling in the new `browser-tck` package.